modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -5,7 +5,7 @@
 [id='olm-installing-from-operatorhub-using-web-console-{context}']
 = Installing from the OperatorHub using the web console
 
-This procedure uses the etcd Operator as an example to install and subscribe to
+This procedure uses the Couchbase Operator as an example to install and subscribe to
```

Comment:
I'd consider making this one generic, too. I'd take the same approach to presenting the CLI and web console commands in both the install and delete topics - use either one or two modules for both.

Reply:
As I noted earlier, I like specific examples. A developer also particularly requested Couchbase be used. Two reasons for giving a real example here. One reason is that not every every Operator works equally well and the developer recommended this one. The other is that I think it's important to show the relationship between the combination of settings: namespace, type of Operator, and Operator.  Not all combinations work together. I'll probably want to go back at some point and fix the generic CLI description.